x/bandtss/keeper/keeper_signing.go

```diff
@@ -129,17 +129,13 @@ func (k Keeper) createSigningRequest(
 			sdk.NewAttribute(types.AttributeKeyCurrentGroupSigningID, fmt.Sprintf("%d", currentGroupSigningID)),
 			sdk.NewAttribute(types.AttributeKeyIncomingGroupID, fmt.Sprintf("%d", incomingGroupID)),
 			sdk.NewAttribute(types.AttributeKeyIncomingGroupSigningID, fmt.Sprintf("%d", incomingGroupSigningID)),
+			sdk.NewAttribute(types.AttributeKeyTotalFee, totalFee.String()),
```

Comment:
Just to be clear so this means that if we do not enter this if if sender.String() != k.authority && currentGroupID != 0 { the total fee in the event will be zero. 
If this is intentional, you can resolve this.

Reply:
Yes, no fee is charged if not fall into those cases.